crates/sui-types/src/sui_sdk_types_conversions.rs

```diff
@@ -134,10 +134,10 @@ impl From<crate::object::Owner> for Owner {
                 initial_shared_version,
             } => Self::Shared(initial_shared_version.value()),
             crate::object::Owner::Immutable => Self::Immutable,
-            // TODO: Corresponding types need to be added to sui-sdk-types.
-            crate::object::Owner::ConsensusV2 {
+            // TODO(ConsensusV2): Corresponding types need to be added to sui-sdk-types.
+            crate::object::Owner::ConsensusAddressOwner {
```

Comment:
Once this is landed i'll make the required changes to the sdk and proto definitions